data/moves.ts

```diff
@@ -13211,7 +13210,7 @@ export const Moves: {[moveid: string]: MoveData} = {
 				break;
 			}
 		},
-		secondary: null,
+		hasSheerForce: true,
```

Comment:
```suggestion
		secondary: null,
		hasSheerForce: true,
```
Every move has `secondary` defined even if it's null, might as well leave it. Sheer Force still handles these weird cases appropriately.